server/smc/smccli/controller/controller.go

```diff
@@ -0,0 +1,59 @@
+package controller
+
+import (
+	"go.dedis.ch/dela/cli"
+	"go.dedis.ch/dela/cli/node"
+)
+
+// smcctl is an initializer with a set of commands for the SMC. It only
+// creates and injects a new SMC instance.
+//
+// - implements node.Initializer
+type smcctl struct{}
+
+// NewSmcController returns a new SMC initializer
+func NewSmcController() node.Initializer {
+	return smcctl{}
+}
+
+// Build implements node.Initializer. In this case we don't need any command.
```

Comment:
This comment, or the function name, is out of date (I don't remember the interface's expectations)